import-export-cli/cmd/aws/init.go

```diff
@@ -0,0 +1,512 @@
+				fmt.Println("Error creating pipe to standard output. (get-export command)", err)
+			}
+
+			err = getExportCmd.Start()
+			if err != nil {
+				fmt.Println("Error starting get-export command.", err)
+			}
+
+			if utils.VerboseModeEnabled() {
+				logsScannerGetExportCmd := bufio.NewScanner(stderr)
+				for logsScannerGetExportCmd.Scan() {
+					fmt.Println(logsScannerGetExportCmd.Text())
+				}
+				if err := logsScannerGetExportCmd.Err(); err != nil {
+					fmt.Println("Error reading debug logs from standard error. (get-export command)", err)
+				}
+			}
+			
+			if utils.VerboseModeEnabled() {
+				outputScannerGetExportCmd := bufio.NewScanner(stdout)
+				for outputScannerGetExportCmd.Scan() {
+					fmt.Println(outputScannerGetExportCmd.Text())
+				}
+				if err := outputScannerGetExportCmd.Err(); err != nil {
+					fmt.Println("Error reading output from standard output. (get-export command)", err)
+				}
+			}
+
+			err = getExportCmd.Wait()
+			if err != nil {
+				fmt.Println("Could not complete get-export command successfully.", err)
+			} 
+			break
+		}
+	}
+	if !found {
+		fmt.Println("Unable to find an API with the name", apiName)
+		os.RemoveAll(tmpDir)
+		os.Exit(1)
+		return err
+	}
+	return nil
+}
+
+// loadDefaultAWSDocFromDisk loads document.yaml stored in HOME/.wso2apictl/document.yaml
+func loadDefaultAWSDoc() (*v2.Document, error) {
+	docData, ok := box.Get("/init/document.yaml")
+	if !ok {
+		return nil, errors.New("Error while retrieving document.yaml")
+	}
+	awsDoc := &v2.Document{}
+	err = yaml.Unmarshal(docData, &awsDoc)
+	if err != nil {
+		return nil, err
+	}
+	return awsDoc, nil
+}
+
+func createAWSDocDirectory(docName string) error {
+	awsDocDirectoryPath := initCmdOutputDir + string(os.PathSeparator) + "Docs"
+	dirPath := filepath.Join(awsDocDirectoryPath, filepath.FromSlash(docName))
+	utils.Logln(utils.LogPrefixInfo + "Creating directory " + dirPath)
+	err := os.MkdirAll(dirPath, os.ModePerm)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+//write document.yaml file
+func writeDocumentFile(docName string, summary string) error {
+	document, err := loadDefaultAWSDoc()
+	docData := &document.Data
+	docData.Name = docName
+	docData.Summary = summary
+	docDataByte, err := yaml2.Marshal(document)
+	if err != nil {
+		return err
+	}
+	apiDocFilePath := filepath.Join(initCmdOutputDir, filepath.FromSlash("Docs" + string(os.PathSeparator) + docName + string(os.PathSeparator) + "document.yaml"))
+	utils.Logln(utils.LogPrefixInfo + "Writing " + apiDocFilePath)
+	err = ioutil.WriteFile(apiDocFilePath, docDataByte, os.ModePerm)
+	return nil
+}
+
+// write AWS API security documents based on APIs security schemes
+func writeAWSSecurityDocs(oas3ByteValue []byte) error {
+	securitySchemes := &v2.SecuritySchemes{}
+	json.Unmarshal(oas3ByteValue, &securitySchemes)
+	schemes := securitySchemes.Components.SecuritySchemes
+	if securitySchemes.ResourcePolicy.Version != "" {
+		docName := "Resource Policy"
```

Comment:
Shall we add these `docName` and `summary` values to the constants file and then use them?

Reply:
aiya but the value of the docName and summary changes depending on the security method. but do you mean like, define the docName and the summary of each method in the constants.go file ?

Reply:
Yes, that would be much clear and consistent with the existing codebase.